.circleci/config.yml

```diff
@@ -80,7 +80,20 @@ commands:
       - store_artifacts: # upload test coverage as artifact.
           path: test-coverage
 
+  # Purpose: Install SFDX, and the release candidate version of the plugin.
+  install_release_candidate:
+    steps:
+      - run:
+          name: "Install SFDX cli"
+          command: npm install -g sfdx-cli
+
+      - run:
+          name: "Install release candidate"
+          command: sfdx plugins:install --force @salesforce/sfdx-scanner@latest-rc
```

Comment:
Sorry I missed this on the last round. Why does this use `--force`?

Reply:
No good reason. I thought I needed it in the past. Turns out we don't.